tools/templates/vfvs_run.py

```diff
@@ -684,7 +1163,7 @@ def create_summary_file(ctx, scenario, collection, scenario_result, output_forma
                 for replica_index in range(scenario['replicas']):
                     record[f"score_replica_{replica_index}"] = ligand['scores'][replica_index]
 
-                df = df.append(record, ignore_index = True)
+            df = df.append(record, ignore_index = True)
```

Comment:
Should merge in the current code (this is backing out a recent change)

Reply:
Will update!